allowed_parameters.py

```diff
@@ -0,0 +1,62 @@
+CHARGEBEE_SUBSCRIPTIONS_ALLOWED_PARAMETERS = [
```

Comment:
I wonder if there's a better way of doing this. Maybe by creating a `ChargebeeRequest` class which is subclassed by `SubscriptionRequest`, and then the allowed params are on the class instead of constant variables like this.